quamash/__init__.py

```diff
@@ -112,17 +81,23 @@ class QThreadExecutor(QtCore.QObject):
 	Same API as `concurrent.futures.Executor`
 
 	>>> from quamash import QThreadExecutor
-	>>> with QThreadExecutor(5) as executor:
+	>>> QtCore = getfixture('qtcore')
+	>>> with QThreadExecutor(QtCore.QThread, 5) as executor:
 	...     f = executor.submit(lambda x: 2 + x, 2)
 	...     r = f.result()
 	...     assert r == 4
 	"""
 
-	def __init__(self, max_workers=10, parent=None):
-		super().__init__(parent)
+	def __init__(self, qthread_class, max_workers=10):
```

Comment:
Yeah, I think there should be a single way of saying which Qt implementation Quamash is to use. `quamash.set_qt` still looks good to me, since we then don't have to infer which Qt package to use. It's comparable to Qt widgets' refusal to be constructed unless there's a QApplication running; the difference in our case is that we don't know the type of QApplication until someone points us to the containing package.